src/layout/msWorkspace/msWorkspace.js

```diff
@@ -244,15 +246,36 @@ var MsWorkspace = class MsWorkspace {
         this.focusTileable(this.tileableList[this.focusedIndex - 1]);
     }
 
-    focusAppLauncher() {
-        this.focusTileable(this.tileableList[this.tileableList.length - 1]);
+    focusAppLauncher(usedHotkey) {
+        if (
+            !this.tileableList ||
+            this.tileableList.length < 2 ||
```

Comment:
Since there is a demand of making the app launcher as an option we should have better test than this